editor/embeddable-autotester.js

```diff
@@ -324,6 +322,13 @@ function autotester(sceneText, nav, sceneName, extraLabels) {
   Scene.prototype.gosub = function scene_gosub(label, inChoice) {
     if (!seen[label.toLowerCase()]) this.oldGosub(label);
   }
+
+  // Params can't be filled with reasonable values (we don't know the types,
+  // let alone additional input constraints), so just treat as return.
+  // Randomtest will test gosubs with a good mix of parameters anyway.
+  Scene.prototype.params = function scene_params(data) {
```

Comment:
Ah, I found it after testing further. The problem is the `gotoSceneLabels` feature of the autotest harness. It starts by searching the whole game for `goto_scene` and `gosub_scene` commands; if it finds them, it adds them to a list of labels to try later.

I've already got some code that excludes this feature from testing when using `[]` or `{}` in the command. I'll add a workaround to exclude parameterized subscenes. Thus params will at least work normally for same-scene `*gosub` commands.